game/scripts/npc/heroes/ember_spirit.txt

```diff
@@ -0,0 +1,11 @@
+"DOTAHeroes"
+{
+  //=================================================================================================================
+  // HERO: Ember Spirit
+  //=================================================================================================================
+  "npc_dota_hero_ember_spirit"
+  {
+    "Ability10"     "special_bonus_attack_damage_75" // replaces special_bonus_attack_damage_25
+    "Ability14"     "special_bonus_spell_amplify_15" // replaces special_bonus_spell_amplify_10
```

Comment:
Ember can still burst people before lvl 25. Keep that in mind. And this should be reverted when I add a new item Horn of Sorcery.

Reply:
What is a horn of sorcery?

Reply:
It will be like negative pipe.

Reply:
What is a horn of sorcery?

Reply:
> 
> 
> It will be like negative pipe (that affects enemies).

Unless this is a really good stat item I doubt ember will buy this item. Ember is not a good aura carrier. If the item on an aura carrier pushes ember over the edge, the item is the problem not the hero. Heroes like OD and Skywrath will abuse the item way better then ember ever could.